src/brokers/consorsbank.js

```diff
@@ -125,10 +125,16 @@ const findDividendShares = textArr => {
   }
   // For older files:
   else {
-    const idxOld = textArr.findIndex(line => line === 'DIVIDENDENGUTSCHRIFT');
-    if (idxOld >= 0) {
-      return parseGermanNum(textArr[idxOld + 1].split(/\s+/)[1]);
+    let lineNumber = textArr.findIndex(line => line === 'DIVIDENDENGUTSCHRIFT');
```

Comment:
Couldn't you use:
```suggestion
    let lineNumber = textArr.indexOf('DIVIDENDENGUTSCHRIFT');
```
This should yield better performance (because thats our biggest issue, right? :D )